src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
+
+  const object = array.reduce((accumulator, current) => {
+    const element = current.split(':');
+
+    if (element.length === 2) {
+      const property = element[0].trim();
+      const value = element[1].trim();
+
+      accumulator[property] = value;
```

Comment:
[CODE STYLE] - It's better to avoid mutations directly inside `reduce`. Instead, you can return a new object with the added property. For example: `return { ...accumulator, [property]: value };`